doc/src/getting_started/aboutus.rst

```diff
@@ -4,7 +4,7 @@ About
 SymPy Development Team
 ----------------------
 
-.. literalinclude:: ../../AUTHORS
+.. literalinclude:: ../../../AUTHORS
```

Comment:
I see, this is the actual AUTHORS file. That's good that it is. But I don't see why it belongs in the documentation.

Reply:
It used to be a curated list of people and what they contributed, but we moved to just copying AUTHORS because curating a list of contributors doesn't scale well. I think it's fine to have a page for it in the docs. That way people who have contributed can get recognition and have somewhere to point to to show that.

Reply:
As per my other comment, I think this would be more appropriate and visible on the website.

Reply:
I can see what you mean, but it would be lot harder technically to include it on the website. The AUTHORS file lives in the repo so it's easy to include it in the docs build, which gets updated with every release. For the website, we would probably need to include it as part of the release script. I'm not opposed to that, but I think it might be out of scope for @nanjekyejoannah's project.

Reply:
That's fine. There are just things in the sphinx docs that don't really fit in the new four categories and there are things in wiki that probably belong in these new categories. I assumed getting this sorted out was part of the project. Maybe it isn't.

Reply:
We decided earlier to leave mostly out of scope moving stuff from the wiki into the docs, because that is a lot more work and just cleaning up what is in the docs already is a bit of work. So we should make the categories based on the idea that things will be moved eventually, but unless there is time at the end, we likely won't be able to do it as part of this project. Moving the authors to the website is probably something we should just open an issue about.

Reply:
Sounds good!